third_party/move/tools/move-unit-test/src/test_reporter.rs

```diff
@@ -122,9 +123,9 @@ impl FailureReason {
 
     pub fn mismatch(
         move_vm_return_values: VMResult<Vec<Vec<u8>>>,
-        move_vm_change_set: VMResult<ChangeSet>,
+        move_vm_change_set: VMResult<Changes<Bytes, Bytes>>,
         stackless_vm_return_values: VMResult<Vec<Vec<u8>>>,
-        stackless_vm_change_set: VMResult<ChangeSet>,
+        stackless_vm_change_set: VMResult<Changes<Bytes, Bytes>>,
```

Comment:
If you keep the old alias, you don't need to change these signatures :)

Reply:
There are a bunch of places ChangeSet is used. If I keep the old alias, I need to change all those places to the new definition. With the new alias for ChangeSet, I only had to change this one place.

Reply:
Not if you use into_custom_effects directly :)